lib/forms/CoBrALab-Mouse-Surgery-Form/index.ts

```diff
@@ -0,0 +1,304 @@
+/* eslint-disable perfectionist/sort-objects */
+
+const { defineInstrument } = await import('/runtime/v1/opendatacapture@1.0.0/core.js');
+const { z } = await import('/runtime/v1/zod@3.23.6/index.js');
+
+function createDependentField<T>(field: T, fn: (treatmentType: string) => boolean) {
+  return {
+    kind: 'dynamic' as const,
+    deps: ['treatmentType'] as const,
+    render: (data: { treatmentType: string }) => {
+      if (fn(data.treatmentType)) {
+        return field;
+      }
+      return null;
+    }
+  };
+}
+
+function createSurgeryDependentField<T>(field: T, fn: (surgeryType: string) => boolean) {
+  return {
+    kind: 'dynamic' as const,
+    deps: ['surgeryType'] as const,
+    render: (data: { surgeryType: string }) => {
+      if (fn(data.surgeryType)) {
+        return field;
+      }
+      return null;
+    }
+  };
+}
+
+export default defineInstrument({
+  kind: 'FORM',
+  language: 'en',
+  tags: ['Surgery', 'Vet Care', 'Wound Treatment', 'Ovariectomy'],
+  internal: {
+    edition: 1,
+    name: 'MOUSE_SURGERY_FORM'
+  },
+  content: {
+    treatmentType: {
+      kind: "string",
+      variant: "select",
+      label: "Select physical intervention",
+      options: {
+        "Surgery": "Surgery",
+        "Wound treatment": "Wound treatment"
+      }
+    },
+    aneglesiaUsed: createDependentField({
+      kind: "boolean",
+      variant: "radio",
+      label: "Aneglesia used"
+    }, (type) => type === "Surgery"),
+
+    aneglesiaType: {
+      kind: "dynamic",
+      deps: ["aneglesiaUsed"],
+      render(data) {
+        if(data.aneglesiaUsed){
+          return {
+            kind: "string",
+            variant: "input",
+            label: "Aneglesia type"
+          }
+        }
+        return null
+      }
+    },
+    aneglesiaVolume: {
+       kind: "dynamic",
+       deps: ["aneglesiaUsed"],
+       render(data) {
+        if(data.aneglesiaUsed){
+          return {
+          kind: "string",
+          variant: "input",
+          label: "Aneglesia volume"
```

Comment:
Should this be only numbers, and specify units?

Reply:
should be number of ml, changing now